core/pkg/resourcehandler/k8sresources.go

```diff
@@ -154,6 +164,15 @@ func (k8sHandler *K8sResourceHandler) GetResources(sessionObj *cautils.OPASessio
 	return k8sResourcesMap, allResources, ksResourceMap, nil
 }
 
+func describeRepositoriesRequired(cloudResources []string) bool {
+	for _, resource := range cloudResources {
+		if strings.Contains(resource, cloudapis.CloudProviderDescribeRepositoriesKind) {
+			return true
+		}
+	}
+	return false
+}
+
```

Comment:
This check can be done within getCloudProviderDescriptionRepositories function. Within the function, you can run `cloudResources := cautils.MapCloudResources(ksResourceMap)` to get the cloudResources.

Reply:
I already have the cloudResources at this stage, I want to check here if they include a specific resource

Reply:
I think it will be cleaner to have this check within the function, but your call.